src/qt/languages/ru-RU.po

```diff
@@ -232,6 +268,9 @@ msgstr "&Перемотка в конец"
 msgid "E&ject"
 msgstr "И&звлечь"
 
+msgid "Eject %s"
+msgstr "И&звлечь"
```

Comment:
```suggestion
msgstr "Извлечь %s"
```
Check original msgid.

Reply:
This line was added for Floppy. To apply the translation for the word "Eject" (Under the line "Export to 86F").